examples/experimental/ProductionMonitoring/StreamTable.md

```diff
@@ -0,0 +1,94 @@
+
+**Note: Column display order is currently non-deterministic** over repeated views of the Panel. If you would like to modify the order, open the StreamTable Panel in a new window as a Board and edit/save a Board from this seed panel. There are two options to achieve this:
+* via the weave.wandb.ai/?exp=... URL
+* via "Open in new tab" arrow button, revealed in the menu when you hover on the right side of a StreamTable panel displayed in the notebok)
+
+![stream_table_from_notebook](../../../docs/assets/stream_table_from_notebook.png)
+
+Continue logging as much data as you like. If you save the StreamTable Panel as a Board, the Board will continue to update as you send more data to the same StreamTable instance.
+
+## StreamTable API Reference
+
+### StreamTable()
+
+Create a StreamTable by providing a table name, with W&B entity (username or team name) and W&B project as optional prefixes to the table name (of the form `entity_name/project_name/table_name`) or separate arguments.
+
+```python
+StreamTable(
+  table_name: str,
+  project_name: typing.Optional[str] = None,
+  entity_name: typing.Optional[str] = None
+)
+```
+
+### .log()
+
+Append rows to the SteamTable. Each row is a dictionary, and `.log()` accepts a single dictionary or a list of dictionaries.
+
+```python
+st = StreamTable("stream_table")
+st.log({"col_A" : 10, "col_B" : "x"})
+st.log([{"col_A" : 20, "col_B" : "y"}, {"col_A" : 30, "col_B" : "z"}])
+```
+
+### .rows()
+
+Add this Weave op to the expression at the top of a Weave Panel to show the contents/actual rows of a StreamTable. Without this op, a StreamTable Panel in the UI will only display the entity, project, and table names (and not the row contents of the StreamTable object).
+
+### .finish()
+
+Wait to show a StreamTable Panel in a notebook UI until all the `.log()` calls have completed / have finished writing to the StreamTable instance (which may mean a longer wait, if e.g. rows are the result of significant computation, running inference, etc).
+ 
+
+## Important usage notes 
+
+Some details to keep in mind for this early iteration of the StreamTables API:
+
+* **columns must match exactly in each log**: each dictionary passed to `.log()` must contain a key for every column in the StreamTable instance. Skipping a key, even if its value is null, or adding a new key/column name after the first log call, may break the StreamTable. 
+* **columns will union across singleton types but not container types**: we strongly recommend keeping the values in one column to one object type (Number, String, image, etc). Logging multiple different singletons to one column will work—that column will become a Union type across the logged instance types. However, logging a container type—say, a List of objects to a String type column—may break the StreamTable.
```

Comment:
Please log a bug with repro and remove this bullet

Reply:
consolidated into one bug + repro notebook: https://wandb.atlassian.net/browse/WB-14552